package.json

```diff
@@ -15,5 +15,12 @@
   "bugs": {
     "url": "https://github.com/erizocosmico/npm-audit-lookout-test/issues"
   },
-  "homepage": "https://github.com/erizocosmico/npm-audit-lookout-test#readme"
+  "homepage": "https://github.com/erizocosmico/npm-audit-lookout-test#readme",
+  "dependencies": {
+    "tough-cookie": "<=2.2.0",
+    "tough-cookie-filestore": "0.0.1"
```

Comment:
A vulnerability has been detected on your `package.json`.

* **Vulnerable dependency:** `tough-cookie`* **Dependency path:** `tough-cookie-filestore>tough-cookie`
* **Dev dependency:** No
* **Severity:** high
* **Installed version:** `0.12.1`
* **Current constraint:** `0.12.1`
* **Vulnerable versions:** `<2.3.3`
* **Patched versions:** `>=2.3.3`
* **Recommendation:** Update to version 2.3.3 or later.

Affected versions of `tough-cookie` are susceptible to a regular expression denial of service.

The amplification on this vulnerability is relatively low - it takes around 2 seconds for the engine to execute on a malicious input which is 50,000 characters in length.

If node was compiled using the `-DHTTP_MAX_HEADER_SIZE` however, the impact of the vulnerability can be significant, as the primary limitation for the vulnerability is the default max HTTP header length in node.

[More info about this vulnerability](https://npmjs.com/advisories/525)